### PR TITLE
kubernetes: Add `Disks` subtab for virtual-machines

### PR DESCRIPTION
**Pull request by mareklibra**
With this patch, the `Disks` subtab is rendered for virtual-machines in the kubernetes view.

The user can click-through to the existing `Volumes` page to review more details.

React component rendering this is shared with pkg/machines.

TODO:
- [x] tests
- [x] PersistentVolume flow type

### CODE REVIEW

**Comment by mareklibra** [comment]
Rebased

**Comment by mareklibra** [comment]
@martinpitt , @jniederm , @andreasn Can you please have a look?

**Comment by mareklibra** [comment]
disks

File: pkg/kubernetes/scripts/virtual-machines/utils.jsx [hunk 1]
```diff
@@ -46,3 +46,25 @@ export function mouseClick(fun) {
...
 }
+
+/**
+ * Used to get sub-property value.
+ *
+ * Example:
+ *   getValueOrDefault(() => myObj.foo.bar, defaultValue) returns value of myObj.foo.bar if path exists or default otherwise
+ *
+ */
+export function getValueOrDefault(accessor, defaultValue) {
```
Comment (martinpitt) [inline pkg/kubernetes/scripts/virtual-machines/utils.jsx]: @bond95, this might be interesting for you as well, in your PR you have this use case.

**Comment by martinpitt** [comment]
Conflict with #8492, needs rebasing now.

**Pull request closed without merging**